openra/tools/uninstall.ps1

```diff
@@ -0,0 +1,42 @@
+<#
+  .SYNOPSIS
+
+  Get the uninstaller location for most Windows software.
+
+  .DESCRIPTION
+
+  This function finds the correct uninstaller given just an application name.
+  Find the name from the Programs and Features Control Panel window.
+
+  .PARAMETER Name
+
+  The name of the application, as found in the Programs and Features applet 
+  in the Control Panel.
+
+  .EXAMPLE
+
+  $tools = Split-Path $MyInvocation.MyCommand.Definition
+  . $tools\uninstall.ps1
+
+  Uninstall-ChocolateyPackage 'foo' 'EXE' '/S' (Get-Uninstaller -Name 'Foo')
+
+  .LINK
+
+  https://github.com/AnthonyMastrean/chocolateypackages/blob/master/helpers/uninstall.ps1
+#>
+function Get-Uninstaller {
+  [CmdletBinding()]
+  param(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Name
+  )
+
+  $local_key     = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*'
+  $machine_key32 = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\*'
+  $machine_key64 = 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\*'

```

Comment:
Yeah... https://msdn.microsoft.com/en-gb/library/aa384129(VS.85).aspx (working on something related to this right now....)